ocs_ci/ocs/resources/pvc.py

```diff
@@ -119,18 +120,25 @@ def resize_pvc(self, new_size, verify=False):
         return True
 
 
-def delete_pvcs(pvc_objs):
+def delete_pvcs(pvc_objs, concurrent=False):
     """
     Deletes list of the pvc objects
 
     Args:
         pvc_objs (list): List of the pvc objects to be deleted
+        concurrent (bool): Determines if the delete operation should be
+            executed with multiple thread in parallel
 
     Returns:
         bool: True if deletion is successful
     """
-    for pvc in pvc_objs:
-        pvc.delete()
+    if concurrent:
+        with ThreadPoolExecutor(max_workers=5) as executor:
```

Comment:
why max_worker is set to 5 here?

Reply:
done - changed to default:
from https://docs.python.org/3/library/concurrent.futures.html:
"If max_workers is None or not given, it will default to the number of processors on the machine, multiplied by 5, assuming that ThreadPoolExecutor is often used to overlap I/O instead of CPU work and the number of workers should be higher than the number of workers for ProcessPoolExecutor."